tutorials/20_score_based_methods_new_features.ipynb

```diff
@@ -0,0 +1,710 @@
+    "\n",
+    "- reverse the diffusion process or perform Langevin dynamics using the estimated scores  \n",
+    "\n",
+    "Note that in this i.i.d. setting, only the `\"sde\"` option is implemented for the moment.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 86,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": []
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "Drawing 100 posterior samples: 100%|██████████| 499/499 [00:13<00:00, 37.00it/s]\n",
+      "Drawing 100 posterior samples: 100%|██████████| 100/100 [00:13<00:00,  7.41it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Build the tall posterior, and sample using sde option\n",
+    "tall_posterior = inference.build_posterior(score_estimator=score_estimator, prior=prior, sample_with=\"sde\")\n",
+    "tall_samples = tall_posterior.sample(sample_shape=(100,),x=x_batch,iid_method=\"auto_gauss\",corrector=\"gibbs\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The argument `iid_method` only applies when `x_o` is a **batch** of observations.  \n",
```

Comment:
to further illustrate the case, maybe do this inference step 2 times, once, with 2 or so trials, once with 20 trials, to see how the posterior becomes more focused. you could switch to a scatter plot in the two-D marginal pairplot to show both posterior in one plot.